test/extensions/filters/http/oauth2/filter_test.cc

```diff
@@ -311,9 +311,6 @@ TEST_F(OAuth2Test, DefaultAuthScope) {
   test_config_ = std::make_shared<FilterConfig>(p, factory_context_.cluster_manager_, secret_reader,
                                                 scope_, "test.");
 
-  // Auth_scopes was not set, should return default value.
-  EXPECT_EQ(test_config_->encodedAuthScopes(), TEST_DEFAULT_SCOPE);
```

Comment:
Removed since it is tested through the location header below and would otherwise require keeping the encodedAuthScopes() method solely for testing purposes.